drivers/interrupt_controller/intc_nxp_irqsteer.c

```diff
@@ -225,20 +225,36 @@
 #include <zephyr/device.h>
 #include <zephyr/devicetree/interrupt_controller.h>
 #include <zephyr/irq.h>
-#include <fsl_irqsteer.h>
 #include <zephyr/cache.h>
 #include <zephyr/sw_isr_table.h>
 #include <zephyr/logging/log.h>
 #include <zephyr/pm/device_runtime.h>
 #include <zephyr/pm/device.h>
-
+#include <zephyr/sys/util.h>
 #include "sw_isr_common.h"
 
-LOG_MODULE_REGISTER(nxp_irqstr);
+LOG_MODULE_REGISTER(nxp_irqsteer);
 
```

Comment:
Is there a reason for this change? All the prefixes in this file are still `nxp_irqster`